### ISSUE TITLE
Question about deserialize to System.Object type

### ISSUE THREAD

**Issue by albertwoo** [Zaid-Ajaj/Fable.SimpleJson#56]
When I convert some string to type like **Map<string, obj> list** the result will be failed. Says some type cannot be converted to System.Object.

Is this mean to be or not? Or should we support this case because sometimes it turns out very useful.

**Comment by Zaid-Ajaj** [comment]
Hi @albertwoo, deserializing something to `obj` doesn't work because we don't know what it the object should become once it has become an `obj` 

> because sometimes it turns out very useful.

Can you please elaborate on cases when this is useful? 🤔

**Comment by albertwoo** [comment]
Hi,

My use case is like:

I fetch some raw data from database through my server dynamicly, so some times the returned result is like:
```json
[
    { "Id": 1,  "Name": "foo1" }
]
```

Sometimes it returns like:
```json
[
    { "Id": 1,  "Age": 20 }
]
```

But from my client I have one table to display any result. So I try to deserialize to **Map<string, obj> list** , So I can get the headers and row values and display very easily without to worry about the actual record and related types.

My simple workaround is like:
```fsharp

                                match jsonData with
                                | JArray ls ->
                                    ls
                                    |> List.map (function
                                        | JObject map ->
                                            map
                                            |> Map.map (fun _ x ->
                                                match x with
                                                | JNumber x -> box x
                                                | JString x -> box x
                                                | JBool x -> box x
                                                | JNull -> box null
                                                | _ -> failwithf "Invalid data: %A" x)
                                        | l ->
                                            failwithf "Invalid data: %A" l)

```

**Comment by Zaid-Ajaj** [comment]
I see your point. However, this is oddly specific and your workaround is the way to go, I would say. 

I'll keep this issue open and will try to implement when time permits 👍

**Comment by albertwoo** [comment]
Thanks :)

**Comment by Zaid-Ajaj** [comment]
Hi @albertwoo, I just published v3.17 of Fable.SimpleJson which includes the ability to deserialize JSON into `obj` :rocket: Please take it for a spin and let me know how it goes :wink: 

Should any problem arise, please reopen the issue 🙏

**Comment by albertwoo** [comment]
cool!!!
